src/main/proto/netflix/titus/agent.proto

```diff
@@ -254,4 +256,17 @@ message ContainerInfo {
 
     // passthrough attributes are an arbitrary set of key / value pairs that are passed from API to executor
     map<string, string> passthroughAttributes = 37;
+
+    // entrypoint and command follow Docker semantics, and can be used to override what is baked into the Docker image:
+    // https://docs.docker.com/engine/reference/builder/#understand-how-cmd-and-entrypoint-interact
+    //
+    // Both CMD and ENTRYPOINT baked into the image can be explicitly cleared with a single empty value (i.e.: `[""]`).
+    message Process {
```

Comment:
Not a big fan of this being called process? Maybe executablConfiguration?

Reply:
I called it `Process` to align with the OCI a bit.

https://github.com/opencontainers/runtime-spec/blob/master/config.md#process